packages/rx_bloc_cli/example/README.md

```diff
@@ -16,7 +16,8 @@
 12. [Push notifications](#push-notifications)
 13. [Social Logins](#social-logins-library)
 14. [Patrol integration tests](#patrol-integration-tests)
-15. [Next Steps](#next-steps)
+15. [Realtime communication](#realtime-communication)
```

Comment:
I believe the `compile.sh` needs to be changed as well, otherwise this change will be reverted with the next releasse

Reply:
Changed in what way? Can you be more explicit?